skbuild/command/egg_info.py

```diff
@@ -45,9 +45,10 @@ def run(self):
             except subprocess.CalledProcessError:
                 sys.stderr.write(
                     '\n\n'
-                    'scikit-build tried to generate a MANIFEST '
-                    'file, but could not because the files to include could '
-                    'not be inferred.\n\n'
+                    'Since scikit-build could not find MANIFEST.in or '
+                    'MANIFEST, it tried to generate a MANIFEST file '
+                    'automatically, but could not because it could not '
+                    'determine which source files to include.\n\n'
                     'The command used was "git ls-tree --name-only -r HEAD"\n'
```

Comment:
It should be whatever the current directory is when the setup command is ran.  Do you think it's really necessary?